src/assisted_test_infra/test_infra/utils/release_image_utils.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import json
+import semver
+
+from assisted_test_infra.test_infra import utils
+
+
+def extract_installer(release_image: str, dest: str):
+    """
+    Extracts the installer binary from the release image.
+
+    Args:
+        release_image: The release image to extract the installer from.
+        dest: The destination to extract the installer to.
+    """
+    logging.info("Extracting installer from %s to %s", release_image, dest)
+    with utils.pull_secret_file() as pull_secret:
+        utils.run_command(
+            f"oc adm release extract --registry-config '{pull_secret}'"
+            f" --command=openshift-install --to={dest} {release_image}"
+        )
+
+
+def extract_version(release_image):
+    """
+    Extracts the version number from the release image.
+
+    Args:
+        release_image: The release image to extract the version from.
+    """
+    logging.info(f"Extracting version number from {release_image}")
+    with utils.pull_secret_file() as pull_secret:
+        stdout, _, _ = utils.run_command(
+            f"oc adm release info --registry-config '{pull_secret}' '{release_image}' -ojson"
+        )
+
+    ocp_full_version = json.loads(stdout).get("metadata", {}).get("version", "")
+    ocp_semver = semver.VersionInfo.parse(ocp_full_version)
+    ocp_version = f"{ocp_semver.major}.{ocp_semver.minor}"
+
+    return ocp_version
+
+
+def extract_rhcos_url_from_ocp_installer(installer_binary_path: str):
```

Comment:
from_release_payload? :)

Reply:
Nope, it's from the installer (which we got from the release payload, but that's beside the point)